packages/wrangler/src/__tests__/api/startDevWorker/ConfigController.test.ts

```diff
@@ -222,10 +228,10 @@ base_dir = \"./some/base_dir\"`,
 			"wrangler.toml": dedent/* toml */ `
                 name = "my-worker"
                 main = "src/index.ts"
+								compatibility_date = \"2024-06-01\"
                 account_id = "1234567890"
             `,
 		});
-		await controller.set({ config: "./wrangler.toml" });
```

Comment:
This is no longer required because we are watching the config file that is being changed above. Previously, when watching was not enabled, we had to trigger this update manually.